services/github/github-common-fetch.js

```diff
@@ -45,10 +45,11 @@ async function fetchRepoContent(
       throw new InvalidResponse({ prettyMessage: 'undecodable content' })
     }
   } else {
-    return serviceInstance._request({
+    const response = await serviceInstance._request({
```

Comment:
Thanks for fixing this bug! I'm a bit surprised this hasn't come up before. I guess this code path is not used. 

One suggestion for consistency with the way most of the shared code is written:

```suggestion
    const { buffer } = await serviceInstance._request({
```

Reply:
At first I was a bit surprised as well but going deeper I realized that block of code was actually never called :) `fetchJsonFromRepo` called `fetchRepoContent` only if `serviceInstance.staticAuthConfigured` was evaluated to true.